build-conflicts.php

```diff
@@ -34,6 +34,11 @@
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromGithubApi;
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromMultipleSources;
 
+use function array_keys;
+use function count;
+use function file_get_contents;
+use function in_array;
```

Comment:
```suggestion
use Psl\Iter;
```

$contains = Iter\contains($array, $value);
$contains_key = Iter\contains_key($array, $key);